draft-ietf-jsonpath-base.md

```diff
@@ -1272,14 +1272,18 @@ rather than the negated existence test `!@.foo` (which yields false if `@.foo` s
 
 The comparison operators `==` and `<` are defined first and then these are used to define `!=`, `<=`, `>`, and `>=`.
 
-When a path resulting in an empty nodelist appears on either side of a comparison:
+When a path resulting in an empty nodelist or a function expression returning `Nothing` appears on either side of a comparison:
 
 * a comparison using the operator `==` yields true if and only if the comparison
-is between two paths each of which result in an empty nodelist.
+is between:
+    * two paths each of which result in an empty nodelist,
+    * two functions each of which returns `Nothing`, or
+    * a path which results in an empty nodelist and a function which returns `Nothing`.
 
 * a comparison using the operator `<` yields false.
 
-When any path on either side of a comparison results in a nodelist consisting of a single node, each such path is
+When any path or function expression on either side of a comparison results in a nodelist consisting of a single node, each such path
```

Comment:
"this side"?

Reply:
Yes, but I think "that side" is better.

Reply:
<img width="856" alt="Screenshot 2023-03-09 at 11 23 26" src="https://user-images.githubusercontent.com/94953/224009268-40136e3f-8450-45af-9341-e9c68f1a2817.png">

Reply:
Yes, but the thread view still shows the commit as it was when the comment was made.